python/lsst/cp/pipe/ptc/cpExtractPtcTask.py

```diff
@@ -714,17 +744,33 @@ def getGainFromFlatPair(self, exposure1, exposure2, correctionType='NONE',
                              "to 'NONE', so a gain value will be estimated without "
                              "corrections." % correctionType)
             correctionType = 'NONE'
-        if region is not None:
-            im1Area = exposure1.getImage()[region].getArray()
-            im2Area = exposure2.getImage()[region].getArray()
-        else:
-            im1Area = exposure1.getImage().getArray()
-            im2Area = exposure2.getImage().getArray()
 
-        const = np.mean((im1Area - im2Area)**2 / (im1Area + im2Area))
-        gain = 1. / const
+        (im1Area, im1MaskVal, im1StatsCtrl, im2Area,
+            im2MaskVal, im2StatsCtrl, mu1, mu2) = imageProperties.values()
```

Comment:
Is there a reason to have this be a dict if it's only used as `imageProperties.values()`?  I think the ordering is guaranteed to be consistent, but a tuple would do the same thing.
If you expect to want to access these by key in the future, then ignore this comment.

Reply:
I'll change it to a tuple. I wasn't sure if a dictionary would be more compact.